hws/README.md

```diff
@@ -49,7 +49,7 @@ Running syntax
 ***optional arguments:***
 | Flag | Description |
 |--|--|
-| -f FILE_PATH | Input steering dump file path |
+| -f FILE_PATH | Input steering dump file path, also the output of the tool will be written to FILE_PATH.parsed |
```

Comment:
'-f FILE_PATH' is not optional parameter.
Please fix it both here in the description (otherwise it doesn't make sense that you have output written to the file.parsed), and in the command (line 45)